smart-cards/src/test/java/gov/va/api/health/smartcards/WellKnownControllerTest.java

```diff
@@ -0,0 +1,69 @@
+package gov.va.api.health.smartcards;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import gov.va.api.health.autoconfig.configuration.JacksonConfig;
+import gov.va.api.health.r4.api.information.WellKnown;
+import gov.va.api.health.smartcards.MetadataProperties.SecurityProperties;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+
+public class WellKnownControllerTest {
+  private WellKnown actual() {
+    return WellKnown.builder()
+        .tokenEndpoint("https://somevasite.va.gov/token")
+        .authorizationEndpoint("https://somevasite.va.gov/authorize")
+        .managementEndpoint("https://somevasite.va.gov/manage")
+        .revocationEndpoint("https://somevasite.va.gov/revoke")
```

Comment:
Auth endpoints are already published by DQ, so it's likely unnecessary to re-publish them here.